platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java

```diff
@@ -128,6 +134,13 @@ public void update(Map<String, Object> props) {
                 ClaimsHandlerManager.MEMBER_USER_ATTRIBUTE);
         String propertyFileLocation = (String) props.get(
                 ClaimsHandlerManager.PROPERTY_FILE_LOCATION);
+        String bindMethod = (String) props.get(ClaimsHandlerManager.BIND_METHOD);
+        String realm = (props.get(ClaimsHandlerManager.REALM) != null) ?
```

Comment:
Sorry. I meant shouldn't we check here and log instead of waiting for the connection to error?